rules/S6896/java/metadata.json

```
@@ -0,0 +1,24 @@
+{
+  "title": "\".add()\" and \".addAll()\" should not be invoked on \"Map\" and \"SequencedMap\" views",
```

My understanding of the title was: I can not use "add" with "Map". I suggest to add "entries":
```suggestion
  "title": "\".add()\" and \".addAll()\" should not be invoked on \"Map\" and \"SequencedMap\" entries views",
```